<!--xml-->
<launch>
  
  
  <node name="Motion_Controller_Node" pkg="icarus_rover_rc" type="Motion_Controller_Node" output="screen">
    <param name="DEBUG_MODE" value="2"/>
    <param name="mc_device" value="/dev/ttyUSB0"/>
    <param name="baudrate" value="115200"/>   
  </node>
</launch>
